pallets/subtensor/tests/mock.rs

```diff
@@ -168,6 +170,11 @@ parameter_types! {
     pub const SenateMaxMembers: u32 = 12;
 }
 
+// Configure collective pallet for Subnet Owners
+parameter_types! { // Based on the number of u16::MAX subnets
+    pub const SubnetOwnersMaxMembers: u32 = u16::MAX as u32;
```

Comment:
@camfairchild this need to change , based on the fact that we have increased the limit on subnets to 31. We also need to account for when users dissolve their networks (subnet 25 recently did this)